<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowConfig_Colors</class>
 <widget class="QWidget" name="ChatWindowConfig_Colors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>406</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KButtonGroup" name="kcfg_chatFontSelection">
     <property name="title">
      <string>Fonts</string>
     </property>
     <property name="current" stdset="0">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel3_2">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Base font in the chat window&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Base &amp;font:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_chatTextColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QRadioButton" name="radioButton">
        <property name="whatsThis">
         <string>Use normal screen font</string>
        </property>
        <property name="text">
         <string>Use system font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Use custom font:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KFontRequester" name="kcfg_chatFont">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Base font in the chat window&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="toolTip">
         <string>The color of normal text</string>
        </property>
        <property name="whatsThis">
         <string>The color of normal text</string>
        </property>
        <property name="text">
         <string>Base &amp;font:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_chatTextColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="kcfg_chatTextColor">
        <property name="toolTip">
         <string>The color of normal text</string>
        </property>
        <property name="whatsThis">
         <string>The color of normal text</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>&amp;Highlight foreground:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_highlightForegroundColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="KColorButton" name="kcfg_highlightForegroundColor">
        <property name="toolTip">
         <string>Foreground color for highlighted messages</string>
        </property>
        <property name="whatsThis">
         <string>Foreground color for highlighted messages</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2_2">
        <property name="toolTip">
         <string>Color for the background of the chat window</string>
        </property>
        <property name="whatsThis">
         <string>Color for the background of the chat window</string>
        </property>
        <property name="text">
         <string>&amp;Background color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_chatBackgroundColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorButton" name="kcfg_chatBackgroundColor">
        <property name="toolTip">
         <string>Color for the background of the chat window</string>
        </property>
        <property name="whatsThis">
         <string>Color for the background of the chat window</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="textLabel1">
        <property name="toolTip">
         <string>Background color for highlighted messages</string>
        </property>
        <property name="whatsThis">
         <string>Background color for highlighted messages</string>
        </property>
        <property name="text">
         <string>Highlight bac&amp;kground:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_highlightBackgroundColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="KColorButton" name="kcfg_highlightBackgroundColor">
        <property name="toolTip">
         <string>Background color for highlighted messages</string>
        </property>
        <property name="whatsThis">
         <string>Background color for highlighted messages</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="toolTip">
         <string>Color used for links in chats</string>
        </property>
        <property name="whatsThis">
         <string>Color used for links in chats</string>
        </property>
        <property name="text">
         <string>&amp;Link color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_chatLinkColor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorButton" name="kcfg_chatLinkColor">
        <property name="toolTip">
         <string>Color used for links in chats</string>
        </property>
        <property name="whatsThis">
         <string>Color used for links in chats</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="color">
         <color>
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Formatting Overrides</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_chatFmtOverride">
        <property name="toolTip">
         <string>Disregard the colors (background and foreground), and font (size and typeface) of incoming messages</string>
        </property>
        <property name="whatsThis">
         <string>Disregard the colors (background and foreground), and font (size and typeface) of incoming messages</string>
        </property>
        <property name="text">
         <string>Ignore peer's formatting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_chatTextColor</tabstop>
  <tabstop>kcfg_chatFont</tabstop>
  <tabstop>kcfg_chatBackgroundColor</tabstop>
  <tabstop>kcfg_chatLinkColor</tabstop>
  <tabstop>kcfg_highlightForegroundColor</tabstop>
  <tabstop>kcfg_highlightBackgroundColor</tabstop>
  <tabstop>kcfg_chatFmtOverride</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_chatFont</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
